Libraries/StyleSheet/processStyle.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict-local
+ */
+'use strict';
+
+import type {ViewStyleProp} from './StyleSheet';
+
+import flattenStyle from './flattenStyle';
+
+export default function processLayoutProps(
+  style: ViewStyleProp,
+): ViewStyleProp {
+  let processedStyle = flattenStyle(style);
+
+  const layoutPropMap = {
```

Comment:
Let's either move this map into the View component (like this PR does https://github.com/facebook/react-native/pull/34567/files) or limit a helper function to only performing the value mapping (like this PR https://github.com/facebook/react-native/pull/34576/). `flattenStyle` is already being imported in View by the pointerEvents PR, and these properties should also be supported for Text, TextInput, and Image